modules/virt-live-migration-metrics.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitor-vmi-migration.adoc
+// * virt/logging_events_monitoring/virt-prometheus-queries.adoc
+
+:_content-type: REFERENCE
+[id="virt-live-migration-metrics_{context}"]
+= Live migration metrics
+
+The following metrics can be queried to show live migration status:
+
+`kubevirt_migrate_vmi_data_processed_bytes`:: The amount of guest operating system (OS) data that has migrated to the new virtual machine (VM). Type: Gauge.
```

Comment:
It seems to me that the metrics queries would be more readable in a table. Of course, this task would be outside the scope of this PR. I will create a small improvement task to reformat these queries if you think that sounds like a good idea.